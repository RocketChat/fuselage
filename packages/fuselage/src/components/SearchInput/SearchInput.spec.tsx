import React from 'react';
import ReactDOM from 'react-dom';

import { SearchInput } from '.';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<SearchInput />, div);
  ReactDOM.unmountComponentAtNode(div);
});
