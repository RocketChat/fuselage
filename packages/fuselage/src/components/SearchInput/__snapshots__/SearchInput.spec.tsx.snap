// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[SearchInput Component] renders without crashing 1`] = `
<body>
  <div>
    <input
      aria-label="search"
      class="rcx-box rcx-box--full rcx-box--animated rcx-input-box--type-search rcx-input-box"
      size="1"
      type="search"
    />
  </div>
</body>
`;
